.ci/pytorch/test.sh

```diff
@@ -459,6 +463,11 @@ test_perf_for_dashboard() {
             "${target_flag[@]}" --"$mode" --"$dtype" --backend "$backend" --disable-cudagraphs "$@" \
             --output "$TEST_REPORTS_DIR/${backend}_no_cudagraphs_${suite}_${dtype}_${mode}_${device}_${target}.csv"
       fi
+      if [[ "$DASHBOARD_TAG" == *freezing_cpu-true* ]]; then
```

Comment:
Please add `freezing_default-true`, `freezing_dynamic-true` and `freezing_aotinductor-true` etc. instead of one `freezing_cpu-true` in here

Reply:
Done, added related config.